packages/amplify-console-hosting/hosting/index.js

```diff
@@ -13,8 +13,9 @@ const HELP_INFO_PLACE_HOLDER =
   'Manual deployment allows you to publish your web app to the Amplify Console without connecting a Git provider. Continuous deployment allows you to publish changes on every code commit by connecting your GitHub, Bitbucket, GitLab, or AWS CodeCommit repositories.';
 const REMOVE_ERROR_MESSAGE = 'There was an error removing the auth resource';
 const HOSTING_NOT_ENABLED = 'Amplify Console hosting is not enabled.';
-const HOSTING_ALREADY_ENABLED =
-  'Amplify Console hosting has already been enabled';
+const HOSTING_ONLY_ENABLED_ONLINE =
```

Comment:
Nit: The name of this `const` could be more clear. What about `HOSTING_ENABLED_IN_CONSOLE`?

Reply:
looks good, changed.